examples/plot_fmri_data_example.py

```diff
@@ -182,8 +190,11 @@ def preprocess_haxby(subject=2, memory=None):
 #############################################################################
 # Now, the clustered inference algorithm which combines parcellation
 # and high-dimensional inference (c.f. References).
-beta_hat, pval_cdl, _, one_minus_pval_cdl, _ = clustered_inference(
-    X, y, ward, n_clusters
+ward_, beta_hat, theta_hat, omega_diag = clustered_inference(
+    X, y, ward, n_clusters, scaler_sampling=StandardScaler()
+)
+beta_hat, pval_cdl, _, one_minus_pval_cdl, _ = clustered_inference_pvalue(
+    X.shape[0], None, ward_, beta_hat, theta_hat, omega_diag
 )
```

Comment:
We'll have to refactor this to a class (same logic as discussed for DCRT etc.)
Are you planing to do that in a forthcoming PR?

Reply:
Yes, I plan to refactor all of this in classes in another PR.